src/components/accessibility/types.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+
+import { A11ySettings } from '../../state/selectors/get_accessibility_config';
+
+interface ScreenReaderTypesProps {
+  chartTypeDescription: string;
+}
+
+/** @internal */
+export function ScreenReaderTypes(props: A11ySettings & ScreenReaderTypesProps) {
+  if (!props.defaultSummaryId) return null;
+  return (
+    <dl id={props.defaultSummaryId}>
```

Comment:
```suggestion
    <dl>
```

The `id` is used below.